### ISSUE TITLE
any news?

### ISSUE THREAD

**Issue by IngveBoe** [Guichaguri/DiscordDJ#73]
i like the bot i have it playing games music from youtube autoplaylist.
i wonder if it can stream radio. m3u8,mp3,ogg,pls stuff like that

**Comment by IngveBoe** [comment]
i love this bot but wonder if it will become more stabel and if we get live streams back?

**Comment by Guichaguri** [comment]
I'll update it eventually. I'm focused in other projects right now.
I also need someone to work with me. I have a lot of ideas, but it takes too much time to make them work properly.

**Comment by IngveBoe** [comment]
I understand:) if only i knew how to code.

**Comment by Guichaguri** [comment]
Yep. I'm back, but this time I'm recoding the whole bot. If you want, I'll give you news when the bot is ready to roll
